lib/cra-template/template/.storybook/main.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  stories: ['../src/**/*.stories.{js|jsx|mdx}'],
```

Comment:
Will we create a TS template too? We could add `ts`/`tsx` here anyway, as renaming a file to `ts` in CRA triggers it to add TypeScript dependencies.

Reply:
Yep we'll create another template for TS.

> as renaming a file to ts in CRA triggers it to add TypeScript dependencies.
Oh I didn't know that! Sure, will add them.